include/libpfc.h

```diff
@@ -12,6 +12,13 @@
 typedef uint64_t  PFC_CFG;
 typedef int64_t   PFC_CNT;
 
+// the next three definitions are the positions of the three fixed counters within
```

Comment:
Added defns for the fixed counters which seem to be "architectural" so they are consistent.